arviz/tests/helpers.py

```diff
@@ -128,6 +132,55 @@ def pyro_centered_schools(data, draws, chains):
     return posterior
 
 
+def tfp_noncentered_schools(data, draws, chains):
+    """Non-centered eight schools implementation for tfp."""
+    del chains
```

Comment:
This is just my curiosity, why do chain needs to explicitly be deleted?

Reply:
Just to follow the other tests, no need in fact.

Reply:
If it has no effect I would delete it. The other tests may need it because of the way the framework works. If tensorflow does not then no need to include it :)

Reply:
this is sort of a funny pattern to allow the function to have the same signature as the others, but make the intention of not using `chains` clear (see bottom of https://github.com/google/styleguide/blob/gh-pages/pyguide.md#214-decision). I think it is also used in the pyro example, where there are no chains. 

I like it in that it limits the number of ignored pylint warnings (my feelings about it are not strong).

Reply:
ah @yaochitc ignore my comment. Thanks all